src/pages/LendingTeams/TeamListing.vue

```diff
@@ -161,6 +161,7 @@
 			>
 				<p
 					class="tw-text-small tw-text-primary tw-flex-1"
+					v-if="shortLoanBecause(team.loanBecause)"
```

Comment:
I think you should just check `team.loanBecause`here instead of pushing it through the truncation method.